app/Services/MentionsService.php

```diff
@@ -698,10 +701,18 @@ protected function mapAttributes()
         $this->text = preg_replace_callback('`\{attribute:(.*?)\}`i', function ($matches) {
             $id = (int) $matches[1];
             $attribute = $this->attribute($id);
+            $fallback = '';
+            if (str_contains($matches[1], '|')) {
+                $fallback = substr($matches[1], strpos($matches[1], "|") + 1);
+            }
 
-            // No entity found, the user might not be allowed to see it
+            // No entity found, the user might not be allowed to see it, if theres a fallback, apply it
             if (empty($attribute)) {
-                $replace = '<i class="unknown-mention unknown-attribute">' . __('crud.history.unknown') . '</i>';
+                if (!$fallback) {
+                    $replace = '<i class="unknown-mention unknown-attribute">' . __('crud.history.unknown') . '</i>';
+                } else {
+                    $replace = '<span class="attribute attribute-mention">' . $fallback . '</span>';
```

Comment:
Replace the classes with the 2 unknown ones so our css wizards can style links that use the fallback :)